jerry-core/ecma/builtin-objects/ecma-builtins.c

```diff
@@ -376,11 +376,11 @@ ecma_instantiate_builtin (ecma_builtin_id_t obj_builtin_id) /**< built-in id */
 
   size_t property_count = ecma_builtin_get_property_count (obj_builtin_id);
 
-  if (property_count > 32)
+  if (property_count > 40)
```

Comment:
Could you add a not about where does this 40 come from?

Reply:
I have added a macro for it